src/main/scala/cromwell/engine/db/package.scala

```diff
@@ -1,10 +1,17 @@
 package cromwell.engine
 
+import cromwell.binding.FullyQualifiedName
+import cromwell.engine.ExecutionIndex.ExecutionIndex
 import cromwell.engine.ExecutionStatus.ExecutionStatus
 
 package object db {
   // CallStatus doesn't exist as an enum already, or reuse ExecutionStatus?
   type CallStatus = ExecutionStatus
+  trait DatabaseKey {
+    val fqn: FullyQualifiedName
+    val index: ExecutionIndex
+  }
+  case class ExecutionDatabaseKey(fqn: FullyQualifiedName, index: ExecutionIndex) extends DatabaseKey
```

Comment:
What have a `trait DatabaseKey`? It doesn't appear to be used anywhere. The `case class ExecutionDatabaseKey` is used extensively, including `DataAccess`, but not the trait.